include/zephyr/bluetooth/buf.h

```diff
@@ -161,6 +161,40 @@ static inline enum bt_buf_type bt_buf_get_type(struct net_buf *buf)
 		->type;
 }
 
+static inline enum bt_buf_type bt_buf_out_type_from_h4_type(uint8_t h4_type)
+{
+	switch (h4_type) {
+	case BT_HCI_H4_CMD:
+		return BT_BUF_CMD;
+	case BT_HCI_H4_ACL:
+		return BT_BUF_ACL_OUT;
+	case BT_HCI_H4_ISO:
+		return BT_BUF_ISO_OUT;
+	default:
+		__ASSERT_NO_MSG(false);
+		return 0xff;
+	}
+}
+
+static inline uint8_t bt_buf_type_into_h4_type(enum bt_buf_type buf_type)
```

Comment:
```suggestion
static inline uint8_t bt_buf_type_to_h4_type(enum bt_buf_type buf_type)
```

Aren't typical transformation function just using `to`? `atoi`, `atol`, etc.?

Reply:
I modeled it after Rust which calls it 'from' and 'into'. I'll change it to 'to'. Should I also change the 'from' to 'to' (and reverse the "operands")? I.e. `bt_buf_h4_type_to_out_type` and `bt_buf_type_to_h4_type`.

Reply:
> I modeled it after Rust which calls it 'from' and 'into'. I'll change it to 'to'. Should I also change the 'from' to 'to' (and reverse the "operands")? I.e. `bt_buf_h4_type_to_out_type` and `bt_buf_type_to_h4_type`.

No opinion :) Either way works. Do we have any pattern in Zephyr that is common?